_docs/index.md

```diff
@@ -3,6 +3,6 @@ title: Docs
 order: 1
 ---
 {% capture include_content %}
-{% include module_inventory_content.md %}
+{% include docs_overview_include.md %}
```

Comment:
See previous note on `include`. There's no significance behind what I was doing here, it was merely an artifact of what was necessary before I moved docs over to a data collection.

Also, in this case, I might opt to copy the content from `docs_overview_include.md` into `_docs/index.md` and remove the `docs_overview_include.md` file since I believe it's only included once, here.